README.md

```diff
@@ -257,7 +257,7 @@ root: ~/<%= @settings["workspace"] %>
 
 ## Starting a session
 
-This will fire up tmux with all the tabs and panes you configured.
+This will fire up tmux with all the tabs and panes you configured, `start` is aliased to `s`.
```

Comment:
Would you mind making this another sentence? I'd be fine with changing the comma to a period, or, in the style of the other comments, "Aliased to `s`."